nixos/modules/services/web-apps/pihole-adminlte.nix

```diff
@@ -0,0 +1,126 @@
+{ pkgs
+, lib
+, config
+, ...
+}:
+
+with lib;
+
+let
+  cfg = config.services.pihole-adminlte;
+  ftlCfg = config.services.pihole-ftl;
+  user = ftlCfg.user;
+  group = ftlCfg.group;
+in
+{
+  options.services.pihole-adminlte = {
+    enable = mkEnableOption "Pi-hole admin dashboard";
+    hostName = mkOption {
+      type = types.str;
+      description = "Domain name for the website.";
+      default = "pi.hole";
+    };
+    package = mkOption {
+      type = types.package;
+      description = "Package for the application";
+      default = pkgs.pihole-adminlte;
+    };
+    dnsServers = mkOption {
+      type = types.str;
+      description = "DNS providers list";
+      default = ''
+        Google (ECS);8.8.8.8;8.8.4.4;2001:4860:4860:0:0:0:0:8888;2001:4860:4860:0:0:0:0:8844
+        OpenDNS (ECS, DNSSEC);208.67.222.222;208.67.220.220;2620:119:35::35;2620:119:53::53
+        Level3;4.2.2.1;4.2.2.2;;
+        Comodo;8.26.56.26;8.20.247.20;;
+        DNS.WATCH;84.200.69.80;84.200.70.40;2001:1608:10:25:0:0:1c04:b12f;2001:1608:10:25:0:0:9249:d69b
+        Quad9 (filtered, DNSSEC);9.9.9.9;149.112.112.112;2620:fe::fe;2620:fe::9
+        Quad9 (unfiltered, no DNSSEC);9.9.9.10;149.112.112.10;2620:fe::10;2620:fe::fe:10
+        Quad9 (filtered + ECS);9.9.9.11;149.112.112.11;2620:fe::11;2620:fe::fe:11
+        Cloudflare;1.1.1.1;1.0.0.1;2606:4700:4700::1111;2606:4700:4700::1001
+      '';
```

Comment:
This list comes hard-coded in [the install script](https://github.com/pi-hole/pi-hole/blob/master/advanced/Scripts/piholeLogFlush.sh)

Reply:
That link is out of date and we should write that into a comment or the description

Reply:
Whoops, wrong link. It comes from here: https://github.com/pi-hole/pi-hole/blob/master/automated install/basic-install.sh#L45-L53
I'll add that link in a comment :+1: